Create cryptojacking.md (#3466)
Codecademy/docs: content/cybersecurity/concepts/cyber-attack/terms/cryptojacking/cryptojacking.md

---
Title: 'Cryptojacking Attack'
Description: 'Cryptojacking is a type of cyberattack where a hacker uses a target’s computing power to mine cryptocurrency without the target’s knowledge or consent.'
Subjects:
  - 'Cybersecurity'
  - 'Computer Science'
Tags:

>   - 'Cybersecurity'
>   - 'Computer Science'

Reviewer: ```suggestion
  - 'Cybersecurity'
  - 'Cryptocurrency'
```

CatalogContent:

>   - 'introduction-to-cybersecurity'
>   - 'paths/cyber-resilience-risk-management'
>   - 'paths/fundamentals-of-cybersecurity'

Reviewer: ```suggestion
  - 'introduction-to-cybersecurity'
  - 'paths/fundamentals-of-cybersecurity'
```
We typically only have two entries for CatalogContent

---

>  Once the device is infected, the attacker can use its resources to mine cryptocurrency, which can be very lucrative. The most commonly mined cryptocurrency is Monero, which is preferred by attackers because it is difficult to trace. Cryptojacking can slow down the infected device and cause it to overheat, which can damage the hardware

Reviewer: ```suggestion

**Cryptojacking** is a type of cyberattack where a hacker uses a target’s computing power to mine cryptocurrency without the target’s knowledge or consent. 

Once the device is infected, the attacker can use its resources to mine cryptocurrency, which can be very lucrative. The most commonly mined cryptocurrency is Monero, which is preferred by attackers because it is difficult to trace. Cryptojacking can slow down the infected device and cause it to overheat, which can damage the hardware.
```

In addition, before "Once the device is infected," we seem to be missing a sentence or two about how the device is infected.

## The Impact of Cryptojacking

> DDoS attacks can be separated into three categories:

Reviewer 2: [suggestion] Per [the style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#links), hyperlink terms that have entries on the Docs' site.
```suggestion
[DDoS attacks](https://www.codecademy.com/resources/docs/cybersecurity/cyber-attack/ddos-attack) can be separated into three categories:
```

> DDoS attacks can be separated into three categories:
> 

Reviewer: ```suggestion
Cryptojacking can have the following impacts on the victim:

```

> 1. *Resource Drain* : Victims experience decreased system performance, increased electricity bills, and premature hardware degradation.
> 2. *Privacy Invasion* :  Cryptojacking invades user privacy as it often involves unauthorized access to computing resources.
> 3. *Reputation Damage* : Organizations can suffer reputation damage if they unwittingly host cryptojacking scripts on their websites or applications.

Reviewer: ```suggestion
- *Resource Drain*: Victims experience decreased system performance, increased electricity bills, and premature hardware degradation.
- *Privacy Invasion*:  Cryptojacking invades user privacy as it often involves unauthorized access to computing resources.
- *Reputation Damage*: Organizations can suffer reputation damage if they unwittingly host cryptojacking scripts on their websites or applications.
```

## How Bug Bounty Hunters Can Help

> 1. *Identify Cryptojacking Scripts →* Look for suspicious scripts or code on websites or within applications you are testing. Cryptojacking scripts often consume excessive CPU resources and can be hidden in the code.

Reviewer 2: [proofreading] See [my comment in line 19](https://github.com/Codecademy/docs/pull/3466/files#r1349782435) for context. 
```suggestion
1. *Identify Cryptojacking Scripts →* Look for suspicious scripts or code on websites or within applications you are testing. Cryptojacking scripts often consume excessive [CPU](https://www.codecademy.com/resources/docs/general/computer-hardware/cpu) resources and can be hidden in the code.
```

> 1. *Identify Cryptojacking Scripts →* Look for suspicious scripts or code on websites or within applications you are testing. Cryptojacking scripts often consume excessive CPU resources and can be hidden in the code.
> 2. *Report Vulnerabilities →* If you discover a cryptojacking vulnerability, report it responsibly to the affected organization or platform, following their established bug bounty program guidelines.
> 3. *Browser Extensions →* Investigate browser extensions that may secretly mine cryptocurrency without user consent. Report any malicious extensions to browser developers or relevant authorities.

Reviewer: ```suggestion
- *Identify Cryptojacking Scripts*: Look for suspicious scripts or code on websites or within applications that are being tested. Cryptojacking scripts often consume excessive CPU resources and can be hidden in the code.
- *Report Vulnerabilities*: Upon discovery of a cryptojacking vulnerability, report it responsibly to the affected organization or platform, following their established bug bounty program guidelines.
- *Browser Extensions*: Investigate browser extensions that may secretly mine cryptocurrency without user consent. Report any malicious extensions to browser developers or relevant authorities.
```

> 3. *Browser Extensions →* Investigate browser extensions that may secretly mine cryptocurrency without user consent. Report any malicious extensions to browser developers or relevant authorities.

Reviewer 2: ```suggestion
3. *Browser Extensions →* Investigate browser extensions that may secretly mine [cryptocurrency](https://www.codecademy.com/resources/docs/general/cryptocurrency) without user consent. Report any malicious extensions to browser developers or relevant authorities.
```

## Preventing Cryptojacking

While bug bounty hunters play a pivotal role in identifying and reporting cryptojacking vulnerabilities, prevention is equally important. Here are some tips for individuals and organizations:

> -*Install Browser Extensions →* Consider installing browser extensions that block cryptojacking scripts and malicious advertisements.
> - *Regularly Update Software →* Keep your operating system, browsers, and security software up-to-date to patch vulnerabilities exploited by cryptojackers.
> - *Educate Employees →* Organizations should educate their employees about the dangers of cryptojacking and enforce strict security policies.
> - *Implement Web Filtering →* Employ web filtering solutions that can identify and block malicious cryptojacking domains and URLs.

Reviewer: ```suggestion
- *Install Browser Extensions*: Consider installing browser extensions that block cryptojacking scripts and malicious advertisements.
- *Regularly Update Software*: Keep operating systems, browsers, and security software up-to-date to patch vulnerabilities exploited by cryptojackers.
- *Educate Employees*: Organizations should educate their employees about the dangers of cryptojacking and enforce strict security policies.
- *Implement Web Filtering*: Employ web filtering solutions that can identify and block malicious cryptojacking domains and URLs.
```
